packages/redux/src/bags/actions/factories/addBagItemFactory.ts

```diff
@@ -89,10 +89,8 @@ const addBagItemFactory =
         ...item,
         productImgQueryParam,
       }));
-      const normalizedBag = normalize(
-        { ...result, items: newItems },
-        { items: [bagItemSchema] },
-      );
+      const ttt = { ...result, items: newItems };
```

Comment:
(something like this, feel free to choose another naming for the const)

```suggestion
      const mergedResult = { ...result, items: newItems };
```